web/utils.py

```
@@ -1,12 +1,16 @@
 import logging
 from datetime import timedelta
+from typing import Any, Dict, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)